homeassistant/components/shelly/__init__.py

```diff
@@ -169,6 +176,39 @@ def shutdown(self):
         self.device.shutdown()
 
 
+class ShellyDeviceRestWrapper(update_coordinator.DataUpdateCoordinator):
+    """Rest Wrapper for a Shelly device with Home Assistant specific functions."""
+
+    def __init__(self, hass, entry, device: aioshelly.Device):
```

Comment:
We need to remove `entry` from params as it's not used.

Reply:
yeah, just noticed